doc/quick-start/README.md

```diff
@@ -64,9 +41,57 @@ docker exec node4 /bin/sh -c "/usr/local/bin/etcdctl --endpoints=\"http://172.20
 ## Validation Test
 
 ```bash
+# This command may not work with a non x86_64 machine (e.g. mac computers with apple silicon)
+RUSTFLAGS="-C target-feature=+crt-static" cargo build --release --bin lock_client --target x86_64-unknown-linux-gnu
```

Comment:
Why validation tests need to build to `x86_64-unknown-linux-gnu` target? I think it need to be able to run on multiple platfroms.

Reply:
The `validation_test.sh` currently uses the `scripts/lock_client` mounted as `/mnt/lock_client` in docker container for validation.
In fact, when the release image is built, `lock_client` is not copied into it because it is unnecessary. As we compile the `lock_client` on the host machine and copy it into `scripts`, it might not work in Mac computers because the rust cross-compilation might be failed.
I think we could modify the CI steps that depends on `validation_test.sh`?

Reply:
How about we put all the build process inside the container?

Reply:
That's what `doc/quick-start/Dockerfile` does. The issue is that we have compiled `lock_client` successfully in docker, but the `validation_test.sh` script does not use it at all.
I planned to export `/mnt` in `$PATH` in container and replace `/mnt/lock_client` with `lock_client` in `validation_test.sh`, then I could copy lock_client into quick-start container's `/usr/local/bin`.

Reply:
done